Tools/unicode/makeunicodedata.py

```diff
@@ -1105,11 +1106,15 @@ def __init__(self, version, cjk_check=True):
                 table[i].east_asian_width = widths[i]
         self.widths = widths
 
-        for char, (p,) in UcdFile(DERIVED_CORE_PROPERTIES, version).expanded():
+        for char, (propname, *propinfo) in UcdFile(DERIVED_CORE_PROPERTIES, version).expanded():
+            if propinfo:
+                # this is not a binary property, ignore it
+                continue
```

Comment:
All the properties defined in `DerivedCoreProperties.txt` happened to be binary until the latest update, so this tool was getting away with the assumption that they are _always_ binary.

As of Unicode 15.1, this file also includes definitions that use the [`Indict_Conjunct_Break` (`InCB`) property, which is an enumeration](https://unicode.org/reports/tr44/#Indic_Conjunct_Break).

With this change, the loop skips over any non-binary properties, since we have nothing to do with them.

Reply:
It seems like it would be safer to explicitly ignore `Indict_Conjunct_Break` rather than throw out everything with a second column.